applications/openshift/kubelet/kubelet_configure_event_creation/tests/ocp4/e2e.yml

```diff
@@ -1,2 +1,7 @@
 ---
-default_result: FAIL
+default_result:
+  worker: FAIL
+  master: FAIL
+result_after_remediation:
+  worker: PASS
+  master: FAIL
```

Comment:
Is this remediation not relevant on the master nodes?

Reply:
It's relevant, but I kept it to the worker nodes to cut down on time.